fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -91,12 +92,13 @@ struct MutationRef {
 	// This is stored this way for serialization purposes.
 	uint8_t type;
 	StringRef param1, param2;
+	uint32_t checksum;
```

Comment:
This modification has impact on backup files. The question is if we want to add checksum for backup's log files: https://github.com/jzhou77/foundationdb/blob/5734002b15ea43ae5d4959ffcf40e14aa76f621f/fdbclient/include/fdbclient/MutationList.h#L113

If we decided to add the checksum, the consumer has to be changed as well, https://github.com/jzhou77/foundationdb/blob/5734002b15ea43ae5d4959ffcf40e14aa76f621f/fdbclient/FileBackupAgent.actor.cpp#L4012

what if in the backup, some files have the checksum and some don't?